src/main/java/com/fundation/webservice/model/ConvertPdfToHtml.java

```diff
@@ -27,8 +27,9 @@
      * @param criteriaConvert
      */
     public void convert(CriteriaConvert criteriaConvert) {
+        CriteriaThumbnailImage criteria = (CriteriaThumbnailImage) criteriaConvert;
         try {
-            Process process = Runtime.getRuntime().exec("3rdparty/" + "poppler/bin/" + "pdftohtml.exe -c -noframes " + criteriaConvert.getSrcPath() + " " + criteriaConvert.getDestPath());
+            Process process = Runtime.getRuntime().exec(Directories.TOOLS_DIR.getDir() + Directories.POPPLER_DIR.getDir() + "pdftohtml.exe -c -noframes " + criteria.getSrcPath() + " " + criteria.getDestPath() + criteria.getName() + criteria.getExt());
```

Comment:
In this line exceeds 120 lines in length please separate them for better code readability.

Reply:
Fixed.